crud/common/utils.lua

```diff
@@ -692,4 +693,20 @@ function utils.check_name_isident(name)
     return true
 end
 
+function utils.update_storage_call_error_description(err, func_name, replicaset_uuid)
+    if err ~= nil then
```

Comment:
It looks like the code will be more readable if we had `if err == nil then return nil end` in the beginning instead of `if err ~= nil`